### ISSUE TITLE
I am unable to get anything in Notable Contribution.

### ISSUE THREAD

**Issue by ninjakx** [lowlighter/metrics#505]
![pic](https://github.com/ninjakx/ninjakx/raw/main/contribution.svg)

**Workflow:**
```

name: Metrics
on:
  # Schedule updates (each hour)
  
  schedule: [{cron: "0 * * * *"}]
  # Lines below let you run workflow manually and on each commit
  workflow_dispatch:
  push: {branches: ["master", "main"]}
jobs:
  github-metrics:
    runs-on: ubuntu-latest
    steps:
      - name: display_information_1
        uses: lowlighter/metrics@latest
        with:
          # Your GitHub token
          token: ${{ secrets.METRICS_TOKEN }}
          user: ninjakx
          # template
          template: classic
          # base
          base: activity, community, repositories, metadata
          # timezone
          config_timezone: Asia/Kolkata
          # filename
          filename: github-metrics.svg
          # programming language
          plugin_languages: yes
          plugin_languages_ignored: html, css
          plugin_languages_skipped: my-testing-repo
          plugin_languages_details: percentage
          plugin_languages_threshold: 2%
          plugin_languages_limit: 15
          plugin_languages_sections: most-used, recently-used
          plugin_languages_indepth: no
          plugin_languages_recent_load: 500
          plugin_languages_recent_days: 7
          commits_authoring: ninjakx
          # coding habits
          plugin_habits: yes
          plugin_habits_from: 200
          plugin_habits_facts: yes
          plugin_habits_charts: yes
          plugin_habits_trim: yes
          # line of code changed
          plugin_lines: yes
          # topics
          plugin_topics: yes
          plugin_topics_sort: stars
          plugin_topics_mode: mastered
          plugin_topics_limit: 0
          # recent activity
          plugin_activity: yes
          plugin_activity_limit: 5
          plugin_activity_days: 14
          plugin_activity_filter: all
          # notable contrib
          plugin_notable: yes
          plugin_notable_filter: stars:>500
          plugin_notable_repositories: no

      #achievements
      - name: display_achievement
        uses: lowlighter/metrics@latest
        with:
          filename: achievements-metrics.svg
          token: ${{ secrets.METRICS_TOKEN }}
          base: ""
          plugin_achievements: yes
          plugin_achievements_display: compact
          config_display: large
          
      # notable contrib
      - name: display_contribution
        uses: lowlighter/metrics@latest
        with:
          filename: contribution.svg
          token: ${{ secrets.METRICS_TOKEN }}
          base: metadata
          repositories_batch: 5
          plugin_notable: yes
          config_padding: 0, 8 + 1%
```

**Token Settings:**

`public_repo, read:org, read:user`

**Comment by ninjakx** [comment]
Can you explain this Norable Contributions thing? Is it same as Organization showing in activity feed?

**Comment by lowlighter** [comment]
It should display organizations where you committed at least once to the main branch of one of its repository.

It might be slightly different than the Github feed because the lattest also displays it if you opened an issue 🙂

___

I haven't looked yet in depth what you reported though, it's possible that's there's an issue

**Comment by ninjakx** [comment]
@lowlighter : I created 2 pull request in covidIndia repository last year, so it should display that org if this is not considering the latest (this yr) contribution.

**Comment by lowlighter** [comment]
@ninjakx Yeah I think it's highly probable that only last year is taken in account.

Will check if it's possible to edit the GraphQL query to load all years

**Comment by lowlighter** [comment]
So I checked but it doesn't seem possible to get past one year without using execessively expansive API requests.

I also tried to use the `commitContributionsByRepository` which seemed promising, but it's limited to 100 repositories without any cursor. 
```graphql
query NotableContributions {
  user(login: "ninjakx") {
    contributionsCollection {
      totalRepositoriesWithContributedCommits
      commitContributionsByRepository(maxRepositories: 100) {
        repository {
          nameWithOwner
        }
      }
    }
  }
}
```

And I couldn't found any mention the repository you mentioned in it though, the only one reported were:
* ninjakx/Coding-practice
* ninjakx/ninjakx
* ninjakx/youtube-video2ppt
* ninjakx/covid_dash
* ninjakx/Restaurant-billing-app

So current implementation seems to be better given that.

I don't think it's really fixable (it's more an API limitation), #516 will display an empty message to say that no recent contribution in organization were found instead of displaying en empty section like you got, it'll at least be more explicit 🙂
